packages/fast-components-react-base/src/radio/README.md

```diff
@@ -0,0 +1,8 @@
+# Radio
+An element that can be turned on and off. *Radio* buttons are almost always listed together in groups.
```

Comment:
Maybe "A radio component is an input element that can be turned on and off"

Reply:
+1 for Chris's comment - radio is really a single-selection model from a group of items

Reply:
going with: "Only one *radio* button within a group of items can be selected at the same time."